addons/survey/models/survey_survey.py

```diff
@@ -50,7 +51,7 @@ def _default_stage(self):
     public_url = fields.Char("Public link", compute="_compute_survey_url")
     # statistics
     tot_sent_survey = fields.Integer("Number of sent surveys", compute="_compute_survey_statistic")
-    tot_start_survey = fields.Integer("Number of started surveys", compute="_compute_survey_statistic")
+    tot_start_survey = fields.Integer("Number of started startedurveys", compute="_compute_survey_statistic")
```

Comment:
startedurveys?

Reply:
Just to be sure it is started, we count only started startedsurveys. Like closed closedsurveys.